composer.lock

```diff
@@ -435,7 +435,7 @@
                 "cache",
                 "caching"
             ],
-            "time": "2017-01-20 09:27:09"
+            "time": "2017-01-20T09:27:09+00:00"
```

Comment:
I'm not sure what goes on here, Composer seems to flip back and forwards between the two formats. Might be an environmental thing?